library-test/build.gradle

```diff
@@ -14,7 +14,6 @@ apply from: "${rootProject.projectDir}/scripts/publish-module.gradle"
 android {
     kotlinOptions.useIR = true
     compileSdkVersion androidCompileSdkVersion
-    buildToolsVersion androidBuildToolsVersion
```

Comment:
I wasn't aware of the default version 👌

Is there a chance that different machines will be using a different version of the build tools if we remove this? Seems not, but asking to double check with you.

Reply:
Project automatically uses the build tools version which is declared by Gradle plugin, it will not use a different version on different machines unless the user specifies another Gradle version. Sample app already ships with `gradlePluginVersion` so the project uses that Gradle. Hence, each Gradle plugin has a minimum supported build tools version which is also a benefit - it means always will use higher build tools therefore one less dependency to update in the app from time to time.

Reply:
Awesome. Thanks for confirming 💯